### PR TITLE
Add mp4 to types.

### PR DESCRIPTION
**Pull request by danada**
We use mp4s for micro-interactions/animations in our projects so it would be helpful to have this type 😄

### CODE REVIEW

File: common/web/base-web/index.d.ts [hunk 1]
```diff
@@ -46,6 +46,11 @@ declare module '*.svg' {
   export default value;
 }
 
+declare module '*.mp4' {
+  const value: string;
+  export default value;
+}
+
```
Comment (chokoswitch) [inline common/web/base-web/index.d.ts]:
```suggestion

  declare module '*.m4a' {
    const value: string;
    export default value
  }
```

File: common/web/base-web/index.d.ts [hunk 2]
```diff
@@ -46,6 +46,15 @@ declare module '*.svg' {
...
+  declare module '*.m4a' {
+    const value: string;
+    export default value
+  }
```
Comment (chokoswitch) [inline common/web/base-web/index.d.ts]:
```suggestion
declare module '*.m4a' {
  const value: string;
  export default value
}
```

File: common/web/base-web/index.d.ts [hunk 3]
```diff
@@ -46,6 +46,15 @@ declare module '*.svg' {
...
 }
 
+declare module '*.mp4' {
+  const value: string;
+  export default value;
+}
+
+declare module '*.m4a' {
+  const value: string;
+  export default value
```
Comment (danada) [inline common/web/base-web/index.d.ts]:
```suggestion
  export default value;
```

Comment (chokoswitch) [reply common/web/base-web/index.d.ts]: Heh - was a bad idea to try to use GitHub to edit the code 😆

Comment (danada) [reply common/web/base-web/index.d.ts]: I'm still not used to the multi-line suggestions 😅

**Pull request merged**